### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by makssisua**
https://makssisua.github.io/react_decompose/

### CODE REVIEW

**Review by OlegShanin — COMMENTED** [review]
Probably all is well

File: src/components/article.jsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import './article.css';
```
Comment (Olexandr6) [inline src/components/article.jsx:2]: The name of component file should start with a capital letter - `Article`

File: src/components/navigation.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './navigation.css';
```
Comment (Olexandr6) [inline src/components/navigation.jsx:2]: The same case... Fix it in all places

File: src/mediaQueries.css [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    margin: 3.5em auto 0;
+    padding: 2em;
+  }
+}
+
+@media (min-width: 800px) {
+  .article {
+    margin: 3.5em auto;
+  }
+}
```
Comment (Olexandr6) [inline src/mediaQueries.css:18]: These queries are related to a particular components, so move them to `Header.css` and `Article.css`

**Review by Olexandr6 — CHANGES_REQUESTED** [review]
Check comments 👇

**Review by Olexandr6 — CHANGES_REQUESTED** [review]
)

**Pull request closed without merging**